README.md

```diff
@@ -29,3 +29,117 @@ The installer scripts also work fine without docker; just run the following two
 
 The unpacking requires recent versions of msitools (0.98) and libgcab
 (1.2); sufficiently new versions are available in e.g. Ubuntu 19.04.
+
+
+# Build instructions for native environments
+
+The following instructions are for setting up MSVC without docker.
+
+## Prerequisites
+
+```bash
+apt-get update
+apt-get install -y wine64-development python msitools python-simplejson python-six ca-certificates winbind
+```
+
+## Installation
+
+We're going to install it into `~/my_msvc` to avoid needing root privileges on a non-contained system.
+You can change the location to wherever you want, even real `/opt/` if you want to deal with chown'ing all files to root before moving
+
+```bash
+# Download and unpack MSVC
+./vsdownload.py --dest ~/my_msvc/opt/msvc
+# Add wrapper scripts, do minor cleanup of the unpacked MSVC installation
+./install.sh ~/my_msvc/opt/msvc
+
+# Custom CMake
+git clone https://gitlab.kitware.com/mstorsjo/cmake
+git checkout 8c1957b8026c60d50a705555a3a212c9217c0a2e
+mkdir build 
+cd build
+../configure --prefix=~/my_msvc/opt/cmake --parallel=$(nproc) -- -DCMAKE_USE_OPENSSL=OFF
+make -j$(nproc)
+make install
+
+# Run wine at least once
+wineserver -k # kills server (optional)
+wineserver -p
+wine64 wineboot
+```
+
+### Setting up your project with CMake
+
+You need to set the path to prioritize our custom CMake, and also to see our MSVC installation
+After that we just run CMake command with a few extra settings:
+
+```bash
+export PATH=~/my_msvc/opt/cmake/bin:$PATH
+export PATH=~/my_msvc/opt/msvc/bin/x64:$PATH
+CC=cl CXX=cl cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_PROGRAMS=ON -DCMAKE_SYSTEM_NAME=Windows -DCMAKE_CROSSCOMPILING=ON
+make
+```
+
+# FAQ
+
+## Does it run on Ubuntu 18.04 LTS?
+
+Yes, but the install scripts won't work because `msitools` is too old. You'll need to install either via Docker or on a real Ubuntu 20.04 LTS machine; and later copy paste the files under `/opt/msvc`
+
+## Does it work with CMake?
+
+Yes, but a custom version is needed or else CMake will complain that `/opt/msvc/bin/x64/cl` can't build a simple program.
+
+It also fixes the `Ninja` Generator which otherwise has trouble finding RC.
+
+## Can it build Debug versions?
+
+Yes, but there may be troubles. From wine errors appearing in the logs/console to problems when trying to launch it because of missing debug redistributables.
+You will also have to install winbind (see next item).
+
+For the best out-of-the-box experience build in Release mode.
+
+## fatal error C1902: Program database manager mismatch; please check your installation
+
+You need winbind: `sudo apt install winbind`
+
+Issue is [being tracked](https://github.com/mstorsjo/msvc-wine/issues/6).
+
+## Does `Ninja` work?
+
+Yes.
+
+##  Does `Unix Makefile` work?
+
+Yes.
+
+## Project gets rebuild from scratch every time
```

Comment:
This issue can skipped from the list, as issue #29 should be fixed now.

Reply:
Done